api/bases/test.openstack.org_tempests.yaml

```
@@ -494,9 +492,31 @@ spec:
                         dir in /etc/test_operator/<file>
                       type: object
                     containerImage:
-                      description: An URL of a tempest container image that should
-                        be used for the execution of tempest tests.
+                      default: ""
+                      description: Container image for AnsibleTest
```

This does not look right. This is `Tempest` CR. 

Maybe something like this?
```suggestion
                      description: A URL of a container image that should be used by the test-operator for tests execution.
```